src/layouts/Login/components/OtpForm.tsx

```diff
@@ -23,6 +23,18 @@ const OTP_TIMER_INTERVAL = 60
 const validateOtp = (value: string) =>
   value.length === 6 || "Please enter a 6 digit OTP."
 
+const getOtpResendMessage = (remainingTime: number): string => {
+  if (remainingTime === 0) {
+    return "Resend OTP"
+  }
+
+  if (remainingTime === 1) {
```

Comment:
hmm, isnt this `if` statement in line 31 redundant? ie. it will still return `Resend OTP in 1s` if you just had the return statement in line 35?

Reply:
Whoops this was a holdover from when we used to spell out seconds - removed in https://github.com/isomerpages/isomercms-frontend/pull/996/commits/3b544476ce349b26bfd28bd6e21ff2ff17b11f08!